parameter_store/admin.py

```diff
@@ -67,7 +65,6 @@ def get_app_list(self, request, app_label=None):
 
 
 param_admin_site = ParamStoreAdmin('param_admin')
-param_admin_site.verbose_name = 'Edge Parameter Store'
```

Comment:
I think this should probably not require changes; I can't recreate errors locally and this didn't seem to bug other users while they were developing (@jananipands unless you noticed issues)